docker-compose.yaml

```diff
@@ -74,6 +74,69 @@ services:
       - elasticsearch
     command: postgres:5432 mysql:3306 minio:9000 inbucket:10080 openldap:389 elasticsearch:9200
 
+  leader:
+    build:
+      context: .
+      dockerfile: ./build/Dockerfile.buildenv
+    working_dir: '/home/mattermost-server'
+    environment:
+      - "MM_SQLSETTINGS_DATASOURCE=postgres://mmuser:mostest@postgres/mattermost_test?sslmode=disable\u0026connect_timeout=10"
```

Comment:
I am wondering if we can just skip this and rely on the config.json value? By default, it is this value, and should work out of the box. For Mattermost staff, they should know to revert back to the docker settings, when running the Make command.

Reply:
Not sure about this, since this supports both environments since the follower and leader services are only used in an HA scenario. So traditional `make run-server or make run` they should just work.

Reply:
Yep, I think it should work even without setting the env var. I'll leave it up to you. 0/5.

Reply:
Agreed that it would be nice to rely on the standard server config file for this, but also 0/5.